aea/connections/stub/connection.yaml

```diff
@@ -7,7 +7,7 @@ license: Apache-2.0
 aea_version: '>=0.3.0, <0.4.0'
 fingerprint:
   __init__.py: QmWwepN9Fy9gHAp39vUGFSLdnB9JZjdyE3STnbowSUhJkC
-  connection.py: QmT6Cs8oTSoZMoL1eLMYZAF1MibXz9pAKcnHTDDmtmXvxj
+  connection.py: QmXZ8QQzdJufp4uqJyQjDDkrNckmTvmuiAiAxpG8vhsomw
```

Comment:
We will need to bump its version and all the references to it..so all agents in packages etc